src/main/java/duke/Command.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+public class Command {
+    protected CommandType commandType;
+    protected String[] inputs;
+    protected boolean isExit;
+
+    /**
+     * Creates a Command object.
+     * @param commandType Type of command given.
```

Comment:
According to the module's coding standard, should there be an empty line between the header comment and the first `@param`? The same suggestion for all other javadocs comments.